Wikipedia/UI-V5/WMFTableOfContentsCell.swift

```diff
@@ -0,0 +1,167 @@
+
+import UIKit
+
+// MARK: - Cell
+public class WMFTableOfContentsCell: UITableViewCell {
+
+    @IBOutlet var sectionSelectionBackground: UIView!
+    @IBOutlet var sectionTitle: UILabel!
+    @IBOutlet var selectedSectionIndicator: UIView!
+    @IBOutlet var indentationConstraint: NSLayoutConstraint!
+    @IBOutlet var leadSectionBorder: UIView!
+    @IBOutlet var sectionBorder: UIView!
+    
+    // MARK: - Init
+    public required init?(coder aDecoder: NSCoder) {
+        section = nil
+        super.init(coder: aDecoder);
+        self.indentationWidth = 10
+    }
+    
+    // MARK: - UIView
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        self.selectedSectionIndicator.alpha = 0.0
+        self.sectionSelectionBackground.backgroundColor = UIColor.wmf_tableOfContentsSelectionBackgroundColor()
+        self.sectionSelectionBackground.alpha = 0.0
+        self.selectionStyle = .None
+    }
+    
+    // MARK: - Accessors
+    public var section: MWKSection?{
+        didSet(section) {
+            self.sectionTitle.text = WMFTableOfContentsCell.titleText(self.section)
+            self.sectionTitle.font = WMFTableOfContentsCell.titleFont(self.section)
+            self.sectionTitle.textColor = WMFTableOfContentsCell.textColor(self.section)
+            self.leadSectionBorder.hidden = !WMFTableOfContentsCell.leadSectionBorderEnabled(self.section)
+            self.sectionBorder.hidden = !WMFTableOfContentsCell.topBorderEnabled(self.section)
+            self.indentationConstraint.constant = WMFTableOfContentsCell.indentationWidth(self.section)
+            self.setNeedsUpdateConstraints()
+        }
+    }
+    
+    public func setSectionSelected(selected: Bool, animated: Bool) {
+        UIView.animateWithDuration(animated ? 0.3 : 0.0) { () -> Void in
+            if(selected){
+                self.sectionSelectionBackground.alpha = 1.0
+            }else{
+                self.sectionSelectionBackground.alpha = 0.0
+            }
+        }
+    }
+    
+    // MARK: - UITableVIewCell
+    public class func reuseIdentifier() -> String{
+        return self.wmf_nibName()
```

Comment:
Being explicit instead of implicit. This is the only reason i usually wrap a single line in another method… to translate to the language of the current domain. In this case, we want a reuseIdentifier, not a nib name.